doc/changelog/11-standard-library/19530-stdlib_repo.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  a new `rocq-init` package for users who don't want to depend on Stdlib
+  (`#19530 <https://github.com/coq/coq/pull/19530>`_,
+  starting to implement `CEP#83 <https://github.com/coq/ceps/pull/83>`_
+  by Pierre Roux).
```

Comment:
Just run `make ci-stdlib` first, and then `dune exec -- coqide` will work as you expected. Same thing for any library in Coq's CI, actually. We should probably extend https://github.com/coq/coq/blob/master/dev/ci/README-developers.md to include this information. (But not in this PR, since this is not particularly specific to this particular package.)

Reply:
> @Zimmi48 Just run `make ci-stdlib` first, and then `dune exec -- coqide` will work as you expected. Same thing for any library in Coq's CI, actually.

That works.  I see that making a trivial change to Coq followed by `make world` deletes stdlib.  That makes some sense, but I hope we soon have a documented way to rebuild both with a single, simple command.  (Also, it seems that developers and users should be able to cause rebuilding multiple packages automatically.  Maybe dune supports that--I have no idea.  This would be a good thing to document for both users and developers.)

It would be good to have a way to prevent stdlib from being deleted by `make world`, which is a big timesaver for development if you know your Coq changes won't affect the library at all and you disable the library version check code.

Regarding the LoadPath, it's strange to see a lot of repeated module names, e.g.

```
Stdlib.Array /home/proj/coq3/_build/install/default/lib/coq/user-contrib/Stdlib/Array
Stdlib.Array /home/proj/coq3/_build/install/default/lib/coq/theories/Array
```

It would be better if the second one could be `Prelude.Array`, if technically possible and not disruptive to users.

These can be addressed outside of this PR.